plugins/modules/packaging/language/npm.py

```diff
@@ -216,13 +216,16 @@ def list(self):
             self.module.fail_json(msg="Failed to parse NPM output with error %s" % to_native(e))
         if 'dependencies' in data:
             for dep in data['dependencies']:
+                dep_version = dep + '@' + str(data['dependencies'][dep]['version'])
+
                 if 'missing' in data['dependencies'][dep] and data['dependencies'][dep]['missing']:
                     missing.append(dep)
                 elif 'invalid' in data['dependencies'][dep] and data['dependencies'][dep]['invalid']:
```

Comment:
```suggestion
            for dep, props in data['dependencies'].items():
                dep_version = dep + '@' + str(props['version'])

                if 'missing' in props and props['missing']:
                    missing.append(dep)
                elif 'invalid' in props and props['invalid']:
```
Not necessary, but a little neater.